src/invoices/InvoicesList/Invoice/utils.js

```diff
@@ -22,5 +22,6 @@ export const createInvoiceLineFromPOL = (poLine, invoiceId) => {
       quantityPhysical * get(poLine, 'cost.listUnitPrice', 0)
       + quantityElectronic * get(poLine, 'cost.quantityElectronic', 0)
     ),
+    accountingNumber: get(poLine, 'vendorDetail.vendorAccount'),
```

Comment:
I thought at this point you have to set account code also.

Reply:
Fixed